### PR TITLE
🌐 Add translations for new batch of strings

### PR DESCRIPTION
**Pull request by newmuis**
Adds translations for five new strings:

- `AMP_STORY_CONSENT_ACCEPT_BUTTON_LABEL`
- `AMP_STORY_CONSENT_DECLINE_BUTTON_LABEL`
- `AMP_STORY_CONSENT_DISMISS_DIALOG_BUTTON_LABEL`
- `AMP_STORY_DOMAIN_DIALOG_HEADING_LABEL`
- `AMP_STORY_DOMAIN_DIALOG_HEADING_LINK`

### CODE REVIEW

File: extensions/amp-story/1.0/_locales/es.js [hunk 1]
```diff
@@ -21,6 +21,21 @@ import {LocalizedStringBundleDef, LocalizedStringId} from '../localization';
  * @const {!LocalizedStringBundleDef}
  */
 export default {
+  [LocalizedStringId.AMP_STORY_CONSENT_ACCEPT_BUTTON_LABEL]: {
+    string: 'Aceptar',
+  },
+  [LocalizedStringId.AMP_STORY_CONSENT_DECLINE_BUTTON_LABEL]: {
+    string: 'Disminución',
```
Comment (Enriqe) [inline extensions/amp-story/1.0/_locales/es.js]: This is not right, it should be "Declinar"

File: extensions/amp-story/1.0/_locales/es-419.js [hunk 2]
```diff
@@ -21,6 +21,21 @@ import {LocalizedStringBundleDef, LocalizedStringId} from '../localization';
  * @const {!LocalizedStringBundleDef}
  */
 export default {
+  [LocalizedStringId.AMP_STORY_CONSENT_ACCEPT_BUTTON_LABEL]: {
+    string: 'Aceptar',
+  },
+  [LocalizedStringId.AMP_STORY_CONSENT_DECLINE_BUTTON_LABEL]: {
+    string: 'Disminución',
```
Comment (Enriqe) [inline extensions/amp-story/1.0/_locales/es-419.js]: Same here. "Declinar"

**Comment by newmuis** [comment]
PTAL @Enriqe

**Comment by Enriqe** [comment]
LGTM

**Pull request merged**